geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaDataStoreSchemaManager.scala

```diff
@@ -137,6 +138,38 @@ trait KafkaDataStoreSchemaManager extends DataStore with LazyLogging {
     }
   }
 
+  def updateKafkaSchema(typeName: String, sft: SimpleFeatureType) = {
```

Comment:
Maybe create some unit tests? I do know that the unit tests for Kafka are buggy and are thus turned off by default, so maybe not. @jnh5y @elahrvivaz Are unit tests worth doing?